CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - `starknet_estimateFee` returns an internal error for v3 transactions with L2 gas `max_price_per_unit` set to zero.
 - `starknet_getCompiledCasm` returns CASM wrapped in a `casm` property.
+- supporting custom versioned constants for multiple Starknet versions.
```

Comment:
This should be added into a new `### Added` section I think. It's not really a bugfix.

Reply:
OK